UnityProject/Assets/Scripts/UI/Systems/Lobby/CharacterSettings.cs

```diff
@@ -179,25 +181,33 @@ public string TheyrePronoun(PlayerScript script)
 
 	public string IsPronoun(PlayerScript script)
```

Comment:
can you call this one ``GetIsPronoun``? At first glance it looks like it returns a bool for the ``Is`` prefix.

Reply:
Actually, no. It is fine as is given the context in which it is used.